cmd/minikube/cmd/start.go

```diff
@@ -1771,6 +1771,21 @@ func validateKubernetesVersion(old *config.ClusterConfig) {
 	}
 	if nvs.GT(newestVersion) {
 		out.WarningT("Specified Kubernetes version {{.specified}} is newer than the newest supported version: {{.newest}}. Use `minikube config defaults kubernetes-version` for details.", out.V{"specified": nvs, "newest": constants.NewestKubernetesVersion})
+		if contains(constants.ValidKubernetesVersions, kubernetesVer) {
```

Comment:
If the user specifies `--force` we should probably skip all this as the point of all these checks are to potentially exit, but if the user uses `--force` we aren't going to exit here anyway. Then we're not wasting time doing internet checks if the user explicitly doesn't care.

Reply:
Noted. Medya said to still do all the checks